Backend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "ido.backend",
+  "version": "0.0.1",
+  "description": "An express API server for iDo Frontend",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/haletothewood/iDO.git"
```

Comment:
code-collective... 💩

Reply:
Yep, forgot to go back and update this. I may actually just rebase the whole branch.